packages/main/src/plugin/dashboard/dashboard-service.ts

```
@@ -52,6 +54,8 @@ export class DashboardService implements Disposable {
 
   @preDestroy()
   dispose(): void {
+    this.disposables.forEach(disposable => disposable.dispose());
```

Biome (2.4.11)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)